mypy.ini

```diff
@@ -1,5 +1,7 @@
 [mypy]
 strict = True
+show_error_codes = True
+disable_error_code = misc
```

Comment:
I really don't want to do that, but it prevents us from having the two files
`file/remote.py`
`container/remote.py`

Should we rename them to `file_remote.py` and `container_remote.py`? (without the subdirectories)

Reply:
Adding in `__init__.py` files fixed it for me. Is that not an option?